timed/mixins.py

```diff
@@ -70,7 +60,7 @@ def get_serializer(self, data, *args, **kwargs):
 
         # enhance entry dicts with model instances
         data = [
-            self.AggregateObject(**{
+            AggregateObject(**{
```

Comment:
That double-dicting is not really necessary. You can get rid of the "outer" dict, it will work the same.

Reply:
I do not see how this should work in this particular case... because what is being done here is to combine two dicts into one before passing it to AggregateObject.

But maybe you meant creation of AggregateObject in tracking/views.py... I removed the outer dict there which was definitely not necessary.

Reply:
No I just meant that pattern -  `**{**{}, **{}}`.